charts/drax/values.yaml

```diff
@@ -0,0 +1,851 @@
+global:
+  # Kubernetes advertise IP
+  #   Description:  Supply the Kubernetes Advertise address of your Kubernetes cluster.
+  #                 This is used by some services that are exposed via a nodePort.
+  #
+  #   Value type:   string
+  #
+  kubeIp: "10.20.20.20"
+
+  # 4G/5G Options
+  #   Description:  Depending on the product bought, you may enable or disable the accompanying features.
+  #
+  #   Value type:   bool
+  #
+  enable4G: false
+  enable5G: true
+
+  # Accelleran License
+  #   Description:  In order to use the Accelleran products, you need to have a license from Accelleran.
+  #                 Please contact Accelleran to get the license.
+  #                 The license file should be saved as a Kubernetes secret. The name of the secret should be
+  #                 supplied here under the "licenseSecretName" field.
+  #                 To create the secret you can use the following command:
+  #                 kubectl create secret generic accelleran-license --from-file=license.crt
+  #
+  # Value type:     enabled: string
+  #                 licenseSecretName: string
+  #
+  accelleranLicense:
+    enabled: "true"
+    licenseSecretName: "accelleran-license"
+
+
+bootstrap:
+  create: true
+  name: "{{ $.Release.Name }}-bootstrap"
+  redis:
+    enabled: true
+    hostname: ""
+    port: 0
+  nats:
+    enabled: true
+    hostname: ""
+    port: 0
+  kafka:
+    enabled: true
+    hostname: ""
+    port: 0
```

Comment:
Could these values be `null` rather than `""` or `0`? I wonder if it might be more clear that we're essentially not setting them and letting the templates use the default values?

Reply:
Behind the scenes `default` is used. So everything that is "empty" can be used. The main reason I did it like this was to indicate the type in the values file.

Reply:
Ah, good idea 👍